Features/Item/InteractableItem.cs

```diff
@@ -1,42 +1,29 @@
-// using System.Text.RegularExpressions;
 using Godot;
 using untitledplantgame.Common;
 using untitledplantgame.EntityStatsDataContainer;
 using untitledplantgame.Inventory.GeneralInventory.UI_ItemCategory;
 
 namespace untitledplantgame.Item;
 
-public partial class InteractableItem : Area2D, IInteractable
+public partial class InteractableItem : AInteractable
 {
 	[Export]
 	private DataContainer _dataContainer;
 
+	[Export]
+	public new string ActionName { get; private set; } = "pickup";
```

Comment:
why is this needed as export?

Reply:
idk why maybe to see it in the inspector uring runtime? I didn't do this